pkg/genre/genre.go

```diff
@@ -48,9 +50,18 @@ func (s *GenreService) PostGenres(ctx echo.Context) error {
 
 }
 
-// func (s *GenreService) GetGenresGenreid(ctx echo.Context) error {
-
-// }
+func (s *GenreService) GetGenresGenreid(ctx echo.Context) error {
+	genreid := ctx.QueryParam("genreid")
+	var genre model.Genre
+	err := s.db.Get(&genre, "SELECT * FROM Genre WHERE id = ?", genreid)
```

Comment:
```suggestion
	err := s.db.Get(&genre, "SELECT * FROM `Genre` WHERE `id` = ?", genreid)
```

必須ではないのですがsql系予約語とテーブル名が同じだとバグるのでテーブル名やカラム名は `` ` ``で囲んでおくのがおすすめです(過去に1敗)

Reply:
a461580 で修正